batchflow/tests/notebooks/torch_benchmark.ipynb

```diff
@@ -0,0 +1,4784 @@
+{
```

Comment:
<p>adding more summarizing cells like this one after each dataframe would be nice</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/668/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>